x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java

```diff
@@ -548,25 +548,26 @@ public void testRemoteMonitoringCollectorRole() {
         });
 
         assertThat(remoteMonitoringAgentRole.indices().allowedIndicesMatcher(GetSettingsAction.NAME)
-                .test(randomFrom(RestrictedIndicesNames.INTERNAL_SECURITY_INDEX, RestrictedIndicesNames.SECURITY_INDEX_NAME)), is(true));
+                .test(randomFrom(RestrictedIndicesNames.SECURITY_INDICES)), is(true));
```

Comment:
See above.
My thought was that _strictly speaking_ this role is supposed to have certain access to the security indices, but no other access to any restricted index.
But that distinction is only conceptual since those two sets are the same.